dart/dynamics/detail/ShapeFrameAspect.hpp

```diff
@@ -97,6 +97,12 @@ struct DynamicsAspectProperties
   /// Secondary coefficient of friction
   double mSecondaryFrictionCoeff;
 
+  /// Primary slip compliance coefficient
+  double mSlipCompliance;
```

Comment:
```suggestion
  double mPrimarySlipCompliance;
```

`mFrictionCoeff` should be renamed to `mPrimaryFrictionCoeff` as well, but it didn't because of the backward compatibility.

Reply:
https://github.com/dartsim/dart/commit/5d81fbbc3a61d493f7f4f201657aa482ed169d2c